packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1020,6 +1020,12 @@ class DefaultTextStyleTransition extends AnimatedWidget {
 /// Using this pre-built child is entirely optional, but can improve
 /// performance significantly in some cases and is therefore a good practice.
 ///
+/// When putting something into [child] instead of [builder], it is not the
```

Comment:
This is basically repeating the information from the previous three paragraphs without really adding anything new.

Reply:
Then maybe I should rephrase the paragraphs above. As is said in the post content of the PR, the original paragraphs can make new learners somehow confused and have wrong thoughts